sogs.ini.sample

```diff
@@ -0,0 +1,102 @@
+; SOGS configuration.  Copy this file to sogs.ini and edit as desired.  It can be left mostly (or
+; entirely) empty: defaults are shown in the comments.
+
+[db]
+
+; The database connection string url for the sogs database where everything is stored.  For sqlite
+; (currently the only supported database) this is `sqlite:///` followed by the path.  E.g.
+; `sqlite:///sogs.db` for sogs.db in the current working directory, or `sqlite:////path/to/sogs.db`
+; for an absolute URL.
+;url = sqlite:///sogs.db
+
+
+[crypto]
+
+; Path to the x25519 private key file; this is a 32-byte file containing the raw private key data.
+;
+;key_file = key_x25519
+
+
+[net]
+
+; Base url for generating self-referring links, for example for the open group URL and QR code shown
+; when accessing via a web browser.  Can be http, https, and may or may not include a port.  This
+; should be the address of a front-end proxy server (e.g. nginx or apache2) that forwards uwsgi
+; requests to pysogs.  Using HTTPS is possible but not particularly recommended: onion requests are
+; already encrypted and so the extra layer of HTTPS encryption does not make requests more secure.
+;
+;base_url = http://example.net
+
+
+; Listening address for oxenmq requests. The socket uses curve encryption using the same x25519 key
+; as the server itself.  Can be specified as a multiline value to listen on multiple
+; addresses/ports.
+;
+;omq_listen = tcp://*:22028
+
+
+; Internal listening address (usually a unix socket path starting with ipc://) for inter-process
+; communication between sogs processes.  This is unauthenticated and so must not be a publicly
+; accessible address!
+;
+;omq_internal = ipc://./omq.sock
+
+
+; Whether we should show recent messages for public rooms on the sogs room page when visited by a
+; browser.
+;
+;http_show_recent = yes
```

Comment:
maybe also an option like `enable_room_viewer` for disabling it entirely as well?

Reply:
Sure, but out of scope for this PR, which is just about moving the settings we currently have.